packages/audio-block/tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+module.exports = {
+    presets: [require('@frontify/fondue/tailwind'), require('../../tailwind.config')],
```

Comment:
```suggestion
    presets: [require('@frontify/fondue/tailwind')],
```

Reply:
the shared tailwind config isn't reachable over that path, [shared readme](https://github.com/Frontify/guideline-blocks/tree/main/packages/shared#using-components)